firefox-ios/Client/Frontend/PasswordGenerator/PasswordGeneratorViewController.swift

```diff
@@ -0,0 +1,205 @@
+        applyTheme()
+    }
+
+    private func setupView() {
+        // Adding Subviews
+        view.addSubview(contentView)
+        contentView.addSubviews(header, descriptionLabel, passwordField, usePasswordButton)
+
+        // Content View Constraints
+        NSLayoutConstraint.activate([
+            contentView.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+                constant: UX.containerPadding),
+            contentView.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                constant: -UX.containerPadding),
+            contentView.topAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.topAnchor,
+                constant: UX.containerPadding),
+            contentView.bottomAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.bottomAnchor,
+                constant: -UX.containerPadding),
+        ])
+
+        // Content View Elements Layout
+        NSLayoutConstraint.activate([
+            header.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            header.trailingAnchor.constraint(
+                equalTo: contentView.trailingAnchor,
+                constant: -UX.headerTrailingPadding),
+            header.topAnchor.constraint(equalTo: contentView.topAnchor),
+            descriptionLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            descriptionLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            descriptionLabel.topAnchor.constraint(
+                equalTo: header.bottomAnchor,
+                constant: UX.containerElementsVerticalPadding),
+            passwordField.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            passwordField.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            passwordField.topAnchor.constraint(
+                equalTo: descriptionLabel.bottomAnchor,
+                constant: UX.containerElementsVerticalPadding),
+            usePasswordButton.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            usePasswordButton.trailingAnchor.constraint(
+                equalTo: passwordField.trailingAnchor),
+            usePasswordButton.topAnchor.constraint(
+                equalTo: passwordField.bottomAnchor,
+                constant: UX.containerElementsVerticalPadding),
+            usePasswordButton.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
+        ])
+    }
+
+    // MARK: - Themable
+    func applyTheme() {
+        let theme = themeManager.getCurrentTheme(for: windowUUID)
+        view.backgroundColor = theme.colors.layer1
+        descriptionLabel.textColor = theme.colors.textSecondary
+        usePasswordButton.applyTheme(theme: theme)
+        passwordField.applyTheme(theme: theme)
+        header.applyTheme(theme: theme)
+    }
+
+    private func buildUsePasswordButton() {
```

Comment:
Should we maybe rename this to `configureUsePasswordButton`? Build implies to me that the UI element has to be created as well.